website/templates/emails/invite_preprints_branded.txt.mako

```diff
@@ -4,11 +4,11 @@
 
 Hello ${fullname},
 
-You have been added by ${referrer.fullname} as a contributor to the preprint "${node.title}" on ${branded_service.name}, powered by the Open Science Framework. To set a password for your account, visit:
+You have been added by ${referrer.fullname} as a contributor to the ${branded_service.preprint_word} "${node.title}" on ${branded_service.name}, powered by the Open Science Framework. To set a password for your account, visit:
 
 ${claim_url}
 
-Once you have set a password, you will be able to make contributions to "${node.title}" and create your own preprints. You will automatically be subscribed to notification emails for this preprint. Each preprint is associated with a project on the Open Science Framework for managing the preprint.  To change your email notification preferences, visit your project or your user settings: ${settings.DOMAIN + "settings/notifications/"}
+Once you have set a password, you will be able to make contributions to "${node.title}" and create your own ${branded_service.preprint_word_plural}. You will automatically be subscribed to notification emails for this ${branded_service.preprint_word}. Each ${branded_service.preprint_word} is associated with a project on the Open Science Framework for managing the ${branded_service.preprint_word}.  To change your email notification preferences, visit your project or your user settings: ${settings.DOMAIN + "settings/notifications/"}
```

Comment:
Soooo. I apologize about this, but after Steve's fantastic suggestion to just get rid of the troublesome words above, making is word plural isn't really necessary. I chatted with Nici and we can just use the singular form here. Which means that we can get rid of all of the code you made to deal with the plurals and just make this a simple change of templates.

I think we've all, er, learned a lot by this journey through possibility, and, um, grown as people?

Reply:
:expressionless: This is literally my face right now.
Yes, I'll go ahead and make the template change. :sweat_smile:

Reply:
I am so sorry.

Reply:
All is well. This is in fact a real learning opportunity. As a college student I was used to not question whatever was in the assignment. By habit, I've just never really thought about asking product to change the requirement. I guess if something is minor and it adds potential clutter to our codebase it is always worth asking whether it is actually needed.